src/cozmo/faces.py

```diff
@@ -188,7 +195,7 @@ class Face(event.Dispatcher):
     which face it is looking at.
     '''
 
-    #: callable: The factory function to return a :class:`ErollNamedFace`
+    #: callable: The factory function to return an :class:`~cozmo.faces.EnrollNamedFace`
```

Comment:
you shouldn't need a fully qualified name if it's in the same scope. problem before was just the typo in `EnrollNamedFace`